api-client/src/mapView/index.ts

```diff
@@ -0,0 +1,17 @@
+interface MapViewViewport {
```

Comment:
I think we do not use interfaces anywhere, so my personal preference would be to use types here as well. Maybe we should add an linting rule for this.

Reply:
we talked about personal preferences :D

Reply:
Thats why I explicitly said personal preference :D I just think we should use either types or interfaces and not both.